{{ cookiecutter.app_name }}/{{ cookiecutter.app_name }}/app.py

```diff
@@ -21,6 +21,23 @@ def startup(self):
 
 def main():
     return {{ app_class_name }}('{{ cookiecutter.formal_name }}', '{{ cookiecutter.bundle }}.{{ cookiecutter.app_name }}')
+{% elif cookiecutter.gui_framework == 'PySide2' %}import sys
+from PySide2 import QtWidgets
+
+
+class {{ app_class_name }}(QtWidgets.QMainWindow):
+    def __init__(self):
+        super().__init__()
+        self.init_ui()
+
+    def init_ui(self):
+        self.setWindowTitle('Hello World')
```

Comment:
This should probably be `{{ cookiecutter.app_name}}`, rather than 'Hello World'.

Reply:
Fixed in 8b9100b399bb975a92fac207670f8eff118a03df